awebox/viz/animation.py

```diff
@@ -114,6 +114,8 @@ def animate_snapshot(plot_dict, cosmetics, fig_name, init_colors=bool(False), pl
     # make text plot
     txt_glb, txt_loc = fill_in_dashboard(fig, plot_dict, index)
 
+    fig.savefig('snapshot.pdf')
```

Comment:
does this always write out a pdf? might not be desired by the user

Reply:
yes. oops... thanks for noticing!